Graphs/AbstractGraph.js

```diff
@@ -556,7 +556,8 @@ export default class AbstractGraph extends React.Component {
             legendWidth,
             legendHeight,
         } = this.getGraphDimension(label, data);
-
+        let legendContentHeight = 4;
```

Comment:
the only problem I have with this is that the height is static not based on the calculated legendHeight